plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -445,26 +444,29 @@ public static function add_component_settings( $settings ) {
 	 *
 	 * @param array $statuses Order statuses.
 	 * @return array formatted statuses.
+	 *
+	 * @deprecated migrate to \Automattic\WooCommerce\Internal\Admin\Settings instead.
 	 */
 	public static function get_order_statuses( $statuses ) {
-		$formatted_statuses = array();
-		foreach ( $statuses as $key => $value ) {
-			$formatted_key                        = preg_replace( '/^wc-/', '', $key );
-			$formatted_statuses[ $formatted_key ] = $value;
-		}
-		return $formatted_statuses;
+		wc_deprecated_function( __CLASS__ . '::' . __FUNCTION__, '9.9.0', '\Automattic\WooCommerce\Internal\Admin\Settings::get_order_statuses' );
+
+		return Settings::get_order_statuses( $statuses );
 	}
 
 	/**
 	 * Get all order statuses present in analytics tables that aren't registered.
 	 *
 	 * @return array Unregistered order statuses.
+	 *
+	 * @deprecated migrate to \Automattic\WooCommerce\Internal\Admin\Settings instead.
 	 */
 	public static function get_unregistered_order_statuses() {
```

Comment:
There's one more call above to `self::get_unregistered_order_statuses()` that we can update to `Settings::get_unregistered_order_statuses()`.

Reply:
`\Automattic\WooCommerce\Internal\Admin\Settings::get_unregistered_order_statuses` is a private non-static one, unfortunately.